manifests/w/Wagnardsoft/DDU/18.0.6.0/Wagnardsoft.DDU.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Wagnardsoft.DisplayDriverUninstaller
+PackageVersion: 18.0.6.0
+PackageLocale: en-US
+Publisher: Wagnardsoft
+PackageName: Display Driver Uninstaller
+License: Freeware
+ShortDescription: Display Driver Uninstaller is a driver removal utility that can help you completely uninstall AMD/NVIDIA/INTEL graphics card drivers and packages from your system, trying to remove all leftovers (including registry keys, folders and files, driver store).

```

```suggestion
ShortDescription: Display Driver Uninstaller is a driver removal utility that can help you completely uninstall AMD/NVIDIA/INTEL graphics card drivers and packages from your system, trying to remove all leftovers (including registry keys, folders and files, driver store).
Moniker: ddu
```
This will allow users to find it using `winget install DDU`